microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -0,0 +1,593 @@
+
+        webViewComponent.GetWebViewWhenReady((IWebView webView) =>
+        {
+            ((IWithVirtualHost)webView).SetVirtualHostMapping("webview2.sample", "Assets\\Html");
+
+            // Navigate to our local content.
+            webViewComponent.Load(new Uri("http://webview2.sample/demo.html"));
+        });
+    }
+}
+```
+
+
+<!-- ------------------------------ -->
+#### Input
+
+There are various ways to handle input in Unity for Mixed Reality applications.
+
+* [Input overview — MRTK2](/windows/mixed-reality/mrtk-unity/mrtk2/features/input/overview) - recommended for MRTK 2.8 applications.
+* [Input — MRTK3](/windows/mixed-reality/mrtk-unity/mrtk3-input/packages/input/overview) - recommended for MRTK 3 applications.
+* [Unity Input System](https://docs.unity3d.com/Packages/com.unity.inputsystem@1.5/manual/index.html)
+
+Regardless of the input system used within your Unity application, interop code between the various application input events and the WebView plugin is required. This means translating those events (such as Pointer events) into a `WebViewMouseEventData` object and then forwarding those events to the plugin via the `IWithMouseEvent` interface:
+
+```C#
+public interface IWithMouseEvents : IWithInputEvents
+{
+    void MouseEvent(WebViewMouseEventData mouseEvent);
+}
+```
+
+WebView2 is unaware of Unity's input system and likely has a different coordinate system than your Unity scene. As a result, when there is a pointer-down event, its coordinates must be translated into the coordinate system of the WebView2 control. Additionally, the pointer-down event needs to be converted into an appropriate `WebViewMouseEventData` event type.
+
+Simple example:
+
+```C#
+using Microsoft.MixedReality.WebView;
+
+public class WebViewExample : MonoBehaviour, IPointerDownHandler
+{
+    // WebView setup steps skipped for brevity
+
+    public void OnPointerDown(PointerEventData eventData)
+    {
+        IWithMouseEvents mouseEventsWebView = webView as IWithMouseEvents;
+
+        // Call hypothetical function which converts the event's x, y into the WebView2's coordinate space.
+        var hitCoord = ConvertToWebViewSpace(eventData.position.x, eventData.position.y);
+
+        WebViewMouseEventData mouseEvent = new WebViewMouseEventData
+        {
+            X = hitCoord.x,
+            Y = hitCoord.y,
+            Type = PointerEvent.PointerDown,
+            Button = PointerButton.Left,
+            TertiaryAxisDeviceType = WebViewMouseEventData.TertiaryAxisDevice.PointingDevice
+        };
+
+        // Propagate the event to the WebView plugin.
+        mouseEventsWebView.MouseEvent(mouseEvent);
+    }
+}
+```
+
+In the above example, pointer-down events are converted into `WebViewMouseEventData` objects and forwarded to the WebView plugin. It is essentially converted into a mouse-down event. In order to effect mouse click events, the pointer-up would need to be handled in a similar fashion.
```

Comment:
```suggestion
In the above example, pointer-down events are converted into `WebViewMouseEventData` objects and forwarded to the WebView plugin. It is essentially converted into a mouse-down event. In order to affect mouse click events, the pointer-up would need to be handled in a similar fashion.
```
tech review this change.
alts:
to emulate mouse click events
to raise mouse click events

Reply:
@michaelfarnsworth -- is "effect" the intended word?  Meaning is unclear, see suggested alt words